src/components/LogInModal/LogInModal.module.css

```diff
@@ -0,0 +1,131 @@
+.modal {
+   width: 100vw;
```

Comment:
Check in this file if you can change any of the colors to the variables that you have.